ImageSharpCompare/ImageSharpCompare.CalcDiffMaskImage.cs

```diff
@@ -0,0 +1,259 @@
+﻿using System;
+using System.IO;
+
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace Codeuctivity.ImageSharpCompare;
+public static partial class ImageSharpCompare
```

Comment:
use `
#pragma warning disable CA1724 // Type names should not match namespaces - this is accepted for now to prevent a breaking change
public static partial class ImageSharpCompare
#pragma warning restore CA1724 // Type names should not match namespaces - this is accepted for now to prevent a breaking change` to silence CA1724 warnings